libraries/chain/include/sysio/chain/abi_serializer.hpp

```diff
@@ -1004,7 +1004,7 @@ void abi_serializer::from_variant( const fc::variant& v, T& o, const Resolver& r
 
 using abi_serializer_cache_t = std::unordered_map<account_name, std::optional<abi_serializer>>;
 using resolver_fn_t = std::function<std::optional<abi_serializer>(const account_name& name)>;
-   
+
```

Comment:
It looks like all the diffs in this file are whitespace, we should set it back to the original so this file isn't altered.